verticapy/toolbox.py

```diff
@@ -188,6 +193,22 @@ def flat_dict(d: dict) -> str:
     return res
 
 
+# ---#
+def erase_space_start_end_in_list_values(L: list):
```

Comment:
@oualib  We can have a more efficient method like below. it will save some performance. 

def erase_space_start_end_in_list_values(L: list):
    L_tmp = [elem for elem in L]
    for idx in range(len(L_tmp)):
        L_tmp[idx]=L_tmp[idx].strip()
    return L_tmp

Reply:
Nice!